site/docs/hive.md

```diff
@@ -79,12 +79,45 @@ In order to query a Hive table created by either of the HiveCatalog methods desc
 ```sql
 SET iceberg.mr.catalog=hive;
 ```
-You should now be able to issue Hive SQL `SELECT` queries using the above table and see the results returned from the underlying Iceberg table. Both the Map Reduce and Tez query execution engines are supported.
+You should now be able to issue Hive SQL `SELECT` queries using the above table and see the results returned from the underlying Iceberg table.
 ```sql
 SELECT * from table_b;
 ```
 
+#### Using Hadoop Catalog
+Iceberg tables created using `HadoopCatalog` are stored entirely in a directory in a filesytem like HDFS. 
+
+##### Create an Iceberg table
+The first step is to create an Iceberg table using the Spark/Java/Python API and `HadoopCatalog`. For the purposes of this documentation we will assume that the fully qualified table identifier is `database_a.table_c` and that the Hadoop Catalog warehouse location is `hdfs://some_bucket/path_to_hadoop_warehouse`. Iceberg will therefore create the table at the location `hdfs://some_bucket/path_to_hadoop_warehouse/database_a/table_c`.
```

Comment:
After the latest changes now it is possible to use Hive `CREATE TABLE` command to create the Hive table and the Iceberg table at the same time. Do we want to document it here, or do we want to do it in a different PR?

Reply:
I think we should add this, but in a separate PR because this one is ready to merge.

Reply:
@pvary - sounds good. What I've done for everything I've documented here is first I've added code to an integration test suite I have that uses the Spark and Hive runtimes on a distributed cluster to really try test everything as an end user would. Can you give me the high level steps needed for what you're talking about above and I'll try reproduce it and can then raise a separate PR to document it? I assume this is related to the first parts of the output format that were merged?

Reply:
This should work now for HiveCatalog:
```
CREATE EXTERNAL TABLE customers (customer_id BIGINT, first_name STRING, last_name STRING)
STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'
```
This should work for other Catalogs:
```
CREATE EXTERNAL TABLE customers (customer_id BIGINT, first_name STRING, last_name STRING)
STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler'
LOCATION '<LOCATION>'
```

The location and the table identifier should be the same ATM.